doc/release-notes/release-notes-1.0.13-rc1.md

```diff
@@ -13,7 +13,7 @@ Cory Fields (1):
 Duke Leto (1):
       Update performance-measurements.sh
 
-Jack Grigg (36):
+Jack Grigg (37):
```

Comment:
This change is because in this release, I manually added a line corresponding to a subsequent commit, but forgot to update this value.